src/main/java/codechicken/nei/NEIClientConfig.java

```diff
@@ -597,6 +600,19 @@ public static boolean isEnabled() {
         return enabledOverride && getBooleanSetting("inventory.widgetsenabled");
     }
 
+    public static boolean isLoaded() {
+        if (configLoaded) {
+            return true;
+        } else {
+            if (pluginLoader != null && !pluginLoader.isAlive()) {
```

Comment:
Instead of checking the thread status, why not set the `configLoaded` variable to true in the thread at the end?

Reply:
Good point. Fixed. I, for some reason, was under the impression that accessing that variable from the thread would require special synchronization.